client/src/pages/learn/javascript-algorithms-and-data-structures/intermediate-algorithm-scripting/index.md

```diff
@@ -5,4 +5,12 @@ superBlock: JavaScript Algorithms and Data Structures
 ---
 ## Introduction to the Intermediate Algorithm Scripting Challenges
 
-This is a stub introduction
\ No newline at end of file
+The following challenges are part of FCC's Intermediate Algorithm Scripting Challenges. These should prepare you to complete the final challenges for the JavaScript Algorithms And Data Structures Certification.
+
+These challenges will allow you to test how much you have learned and which parts you may need to brush up again before starting the projects.
```

Comment:
is to "bush up" a common enough idiom to be used here?  Can we changes this to something more clear to non-native english speakers, maybe just "review again"?

Reply:
Let's get @RandellDawson 's opinion. I'm fine with either

Reply:
That is fine.

Reply:
@cmccormack You want me to change it?

Reply:
I would go ahead and implement @cmccormack's suggestion.  That is the only thing holding up this PR from being merged.

Reply:
Going ahead

Reply:
@RandellDawson Fixed.